pcsx2/Counters.h

```diff
@@ -85,17 +85,17 @@ struct SyncCounter
 //------------------------------------------------------------------
 // SPEED HACKS!!! (1 is normal) (They have inverse affects, only set 1 at a time)
 //------------------------------------------------------------------
-#define HBLANK_COUNTER_SPEED	1 //Set to '3' to double the speed of games like KHII
+#define HBLANK_COUNTER_SPEED	2 //Set to '3' to double the speed of games like KHII
 //#define HBLANK_TIMER_SLOWDOWN	1 //Set to '2' to increase the speed of games like God of War (FPS will be less, but game will be faster)
 
 //------------------------------------------------------------------
 // NTSC Timing Information!!! (some scanline info is guessed)
 //------------------------------------------------------------------
 #define FRAMERATE_NTSC			29.97 // frames per second
 
-#define SCANLINES_TOTAL_NTSC	525 // total number of scanlines
+#define SCANLINES_TOTAL_NTSC	263 // total number of scanlines
 #define SCANLINES_VSYNC_NTSC	3   // scanlines that are used for syncing every half-frame
-#define SCANLINES_VRENDER_NTSC	240 // scanlines in a half-frame (because of interlacing)
+#define SCANLINES_VRENDER_NTSC	120 // scanlines in a half-frame (because of interlacing)
```

Comment:
This is wrong though, as a "half-frame" of active video in NTSC is definitely 240 vlines.